### ISSUE TITLE
BUG?: error while symlinking

### ISSUE THREAD

**Issue by nickeubank** [andrewrynhard/atom-latex-plus#58]
Getting the following error:

```
Error while symlinking /users/nick/projects/pk_schooling/docs/v2/eubank_pk_ethnic_sorting.synctex.gz to /users/nick/projects/pk_schooling/docs/v2/.latex/eubank_pk_ethnic_sorting.synctex.gz

Error: EINVAL: invalid argument, readlink '/Users/nick/projects/pk_schooling/docs/v2/eubank_pk_ethnic_sorting.synctex.gz'
```

Though build _seems_ to be doing fine...

(OSX 10.11.4)

Possibly #48?

Here are folder contents:

![screen shot 2016-05-16 at 9 24 24 am](https://cloud.githubusercontent.com/assets/9683693/15295964/0dba49bc-1b48-11e6-9517-1c94c4400050.png)
![screen shot 2016-05-16 at 9 24 32 am](https://cloud.githubusercontent.com/assets/9683693/15295970/0fbe6982-1b48-11e6-9498-d82d5c50d1d7.png)

**Comment by andrewrynhard** [comment]
Yes this seems related to #48. We will be making some changes (hopefully soon) that should resolve this.

**Comment by andrewrynhard** [comment]
Closed by #71
